css/popup.scss

```diff
@@ -54,311 +35,11 @@ html {
 }
 
 body {
+  line-height: 1;
   font-family: "Helvetica Neue", sans-serif !important;
   font-weight: normal;
   border: 0;
   margin: 0;
   padding: 0;
 }
```

Comment:
Everything except for font-family here needs to go in a normalize file. The normalize file will have its own line-height rule: 
https://github.com/necolas/normalize.css/blob/master/normalize.css#L12